pkgs/development/python-modules/jax/default.nix

```diff
@@ -19,16 +19,16 @@ let
 in
 buildPythonPackage rec {
   pname = "jax";
-  version = "0.3.4";
+  version = "0.3.5";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "google";
     repo = pname;
-    rev = "jax-v${version}";
-    sha256 = "sha256-RZqSJP2vtt8U6nmftV2VzfkMGkkk3100QqsjI7PpQbc=";
+    rev = "refs/tags/jax-v${version}";
```

Comment:
Is there any reason to use `refs/tags/foobar` instead of simply `foobar`?

Reply:
cc @ryantm

Reply:
@samuela I don't know, it's just doing whatever the updateScript wanted, lets ask @jonringer

Reply:
@samuela Yes, there is a good reason: https://github.com/NixOS/nixpkgs/commit/7f5915b04801f383a3504a011601e2fb09dfedf8